source-postgres-batch/driver.go

```diff
@@ -559,6 +556,9 @@ func (c *capture) poll(ctx context.Context, bindingIndex int, tmpl *template.Tem
 			return nil, fmt.Errorf("error emitting checkpoint: %w", err)
 		}
 
+		if len(cursorValues) != len(cursorNames) {
```

Comment:
nit: perhaps a comment that this addresses the "empty result set" case?

Reply:
Done